packages/datagateway-dataview/src/views/table/investigationTable.component.tsx

```diff
@@ -246,7 +246,16 @@ const InvestigationTable = (
         {
           icon: <AssessmentIcon />,
           label: t('investigations.instrument'),
-          dataKey: 'instrument.name',
+          dataKey: 'investigationInstruments.instrument.name',
+          cellContentRenderer: (cellProps: TableCellProps) => {
+            const investigationData = cellProps.rowData as Investigation;
+            if (investigationData?.investigationInstruments?.[0]?.instrument) {
+              return investigationData.investigationInstruments[0].instrument
```

Comment:
I think if you wanna get this line covered, you'll have to change the test data from this:
```
state.dgcommon.data = [
    {
        ....
        instrument: {
            name: 'LARMOR',
        },
    }
]
```
to something like this (I borrowed it from `investigations.text.tsx`):
```
state.dgcommon.data = [
    {
        ....
        investigationInstruments: [
            {
                id: 3,
                instrument: {
                    id: 4,
                    name: 'LARMOR',
                },
            }
        ],
    }
]
```

Reply:
I definitely would've forgotten to sort out the snapshot if it wasn't for the reminder :)